tasks/config.yml

```diff
@@ -35,6 +35,12 @@
     mode: '0600'
   notify: restart bigbluebutton
 
+- name: Encode welcome message and footer message to UTF8
+  command: native2ascii -encoding UTF8 /etc/bigbluebutton/bbb-web.properties /etc/bigbluebutton/bbb-web.properties
+  become: true
+  notify:
```

Comment:
@mschwrdtnr
The solution that comes to my mind would be the following.

Get the md5 of the file that is going to be edited.
Run the encode step.
Check the file again if changed, then restart.

Something like that could work.

Reply:
That would be one way. Although in my opinion it would be enough to create a handler which only runs when the file has changed...